website/client/src/components/chat/chatMessages.vue

```diff
@@ -331,6 +326,9 @@ export default {
       if (profile && !profile.rejected) {
```

Comment:
here (lines 326-331) basically the same thing happens in both cases, if profile is rejected or not because `profile._id` and `memberId` should be the same value